numpy/lib/tests/test_function_base.py

```diff
@@ -341,6 +341,12 @@ def test_upcasting(self):
             w = np.array([[1,2],[3,4]], dtype=wt)
             assert_equal(np.average(a, weights=w).dtype, np.dtype(rt))
 
+    def test_object_dtype(self):
+         a = np.array([decimal.Decimal(x) for x in range(10)])
++        w = np.array([decimal.Decimal(1) for _ in range(10)])
```

Comment:
What is this plus doing here? Looks like you tried to copy a patch from the PR and paste it.